upload_components/upload.sh

```diff
@@ -17,17 +33,25 @@ for ITEM in "${DIRECTORIES[@]}"; do
         NAME=$(basename "$(realpath "${FULL_PATH}")")
     fi
 
+    echo "Creating namespace ${NAMESPACE}"
+    python3 -m idf_component_manager create-remote-namespace \
```

Comment:
This is a temporary thing, right? In the future the namespace will be created at the time the user gets registered?

Reply:
Yes it is. In future we will have more fine-grained access control on the namespace. Each `write:component` scope token should only have access to his own namespace.

For now, everyone's using admin token and can create a new component only if the namespace is created. (I think we only have namespace "espressif"?)

Reply:
For now it's possible to create a new namespace with the admin panel, later it will also be possible to do it with user-facing UI.

It's a rare function, so I think it should not be exposed as a CLI command at all. And we should not create namespaces automatically with this CI action.